samples/apps/copilot-chat-app/webapi/Model/ChatSession.cs

```diff
@@ -28,10 +28,17 @@ public class ChatSession : IStorageEntity
     [JsonPropertyName("title")]
     public string Title { get; set; }
 
+    /// <summary>
+    /// Timestamp of the message.
+    /// </summary>
+    [JsonPropertyName("timestamp")]
+    public DateTimeOffset Timestamp { get; set; }
```

Comment:
This new property is not being used anywhere.  Unclear how this is going to be used.

Reply:
We plan on using the timestamp property in a different UI/application. Users will see a list of chat sessions and pick one that they would like to use. 

The WebAPI project is fantastic, and we'd like to use it in multiple applications. The hope is that it stays decoupled from the existing UI sample, as it is today, since it provides a lot of great code and good patterns.

Reply:
Glad to hear you are creating another application with the WebAPI!

Some feedback:
If you need to have a time property in the chat sessions, we suggest renaming the property to `CreatedOn` if it's going to be read only. If you are planning to modify the timestamp, consider renaming it to `LastModified` and include the logic that will update the property.

Reply:
That makes a lot of sense. :)

Timestamp was chosen to be more consistent with the ChatMessage object but CreatedOn is more explicit and seems like a better option. We can make the update if you think it is a reasonable PR. Seems like it would only help.

Reply:
Yes, totally agree on keeping it consistent with the `ChatMessage`. I didn't want this PR to include the changes to `ChatMessage` because it will involve changes to the WebApp too. But if you are inclined, feel free to make changes to `ChatMessage` too (and the WebApp).